files/initramfs-tools/union-mount.j2

```diff
@@ -151,16 +165,21 @@ mount --bind ${rootmnt}/host/$image_dir/boot ${rootmnt}/boot
 
 ## Mount loop device or tmpfs for /var/log
 if $logs_inram; then
-  # NOTE: some platforms, when reaching initramfs stage, have a small
-  #       limit of mounting tmpfs partition, potentially due to amount
-  #       of RAM available in this stage. e.g. Arista 7050-qx32[s] and 7060-cx32s
-  set_tmpfs_log_partition_size
-  mount -t tmpfs -o rw,nosuid,nodev,size=${varlogsize}M tmpfs ${rootmnt}/var/log
-  [ -f ${rootmnt}/host/disk-img/var-log.ext4 ] && rm -rf ${rootmnt}/host/disk-img/var-log.ext4
+    # NOTE: some platforms, when reaching initramfs stage, have a small
```

Comment:
Reading this comment I feels it should become
```
NOTE: Some platforms have a small flash and therefore the log partition takes valuable space.
      To improve the longevity of these devices storing the logs in memory will permit more
      SONiC image growth before being bottlenecked.
```

Reply:
The statement means that we want to limit the size we give to ramdisk. I made it clearer. I also added your statement.